tests/test_main.py

```diff
@@ -289,67 +292,57 @@ def test_initialize_github_error(
         mock_report_failure.assert_called_with(self.mock_config)
 
     @mock.patch(PATH + "handle_msg")
-    @mock.patch(PATH + "fedmsg")
-    def test_listen_no_handlers(self, mock_fedmsg, mock_handle_msg):
+    @mock.patch(PATH + "load_config")
+    def test_listen_no_handlers(self, mock_load_config, mock_handle_msg):
         """
         Test 'listen' function where suffix is not in handlers
         """
         # Set up return values
-        mock_fedmsg.tail_messages.return_value = [
-            ("dummy", "dummy", "mock_topic", self.old_style_mock_message)
-        ]
+        mock_load_config.return_value = self.mock_config
 
         # Call the function
-        m.listen(self.mock_config)
+        self.old_style_mock_message.topic = "mock_topic"
+        m.callback(self.old_style_mock_message)
 
         # Assert everything was called correctly
         mock_handle_msg.assert_not_called()
 
     @mock.patch(PATH + "handle_msg")
     @mock.patch(PATH + "issue_handlers")
-    @mock.patch(PATH + "fedmsg")
-    def test_listen_no_issue(self, mock_fedmsg, mock_handlers_issue, mock_handle_msg):
+    @mock.patch(PATH + "load_config")
+    def test_listen_no_issue(
+        self, mock_load_config, mock_handlers_issue, mock_handle_msg
```

Comment:
`issue_handlers` is a `dict` -- we should probably be using [`@mock.patch.dict()`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict) to patch it.  (See lines [401-403](https://github.com/release-engineering/Sync2Jira/blob/fb8ca0eb9447fedc7f0fa90976fe4177fab591da/tests/test_main.py#L401-L403) and [421-423](https://github.com/release-engineering/Sync2Jira/blob/fb8ca0eb9447fedc7f0fa90976fe4177fab591da/tests/test_main.py#L421-L423).)